docset/winserver2022-ps/failoverclusters/Set-ClusterAffinityRule.md

```diff
@@ -0,0 +1,227 @@
+---
+description: Set-ClusterAffinityRule
+external help file: ClusterAffinityRule.cdxml-help.xml
+Module Name: FailoverClusters
+ms.date: 07/27/2023
+online version: https://learn.microsoft.com/powershell/module/failoverclusters/set-clusteraffinityrule?view=windowsserver2022-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+title: Set-ClusterAffinityRule
+---
+
+# Set-ClusterAffinityRule
+
+## SYNOPSIS
+Enabled or Disable an affinity rule, and update the rule type.
+
+## SYNTAX
+
+### Query (cdxml) (Default)
+
+```
+Set-ClusterAffinityRule [[-Name] <String[]>] [-RuleType <RuleType>] [-Enabled <UInt32>]
+ [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>] [-AsJob] [-PassThru] [<CommonParameters>]
+```
+
+### InputObject (cdxml)
+
+```
+Set-ClusterAffinityRule -InputObject <CimInstance[]> [-RuleType <RuleType>] [-Enabled <UInt32>]
+ [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>] [-AsJob] [-PassThru] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet is used to Enabled or Disable an affinity rule. This can also be used to change the
+RuleType.
+
+## EXAMPLES
```

Comment:
Can we add examples for disabling a rule and changing the rule type? It would also be useful to show how you can use this cmdlet in the pipeline.